src/components/ComputeModal.js

```diff
@@ -686,6 +715,8 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
         hasGpu: !!gpuConfig,
         gpuType: gpuConfig?.gpuType || defaultGpuType,
         numGpus: gpuConfig?.numOfGpus || defaultNumGpus,
+        autopauseEnabled: autopauseConfig?.enabled ?? true,
+        autopauseThreshold: autopauseConfig?.enabled ? (autopauseConfig.threshold ?? 0) : defaultAutoPauseThreshold,
```

Comment:
I like @dmohs' [comment](https://github.com/DataBiosphere/terra-ui/pull/2818/files#diff-149d7ad595448fb2f6bedcff6dde5f4cbcccd8d5148fed8820ce88190a3bc38bR710) for the line above; i.e. using `Utils.switchCase` to flatten the nested ternaries, which to me makes it easier to read:

```javascript
const autopauseThreshold = Utils.switchCase(autopauseConfig?.enabled,
  [undefined, () => defaultAutoPauseLength],
  [false, () => 0],
  [true, () => autopauseConfig.threshold]
)
```

P.S. `Utils` methods in `utils.js` are in general worth getting familiar with as we tend to utilize them a lot in this codebase 😊